Sources/AudioKit/Nodes/Mixing/Mixer.swift

```diff
@@ -4,10 +4,14 @@ import AVFoundation
 import CAudioKit
 
 /// AudioKit version of Apple's Mixer Node. Mixes a varaiadic list of Nodes.
-public class Mixer: Node, Toggleable {
+public class Mixer: Node, Toggleable, NamedNode {
+
     /// The internal mixer node
     fileprivate var mixerAU = AVAudioMixerNode()
 
+    /// Name of the mixer
+    open var name: String?
```

Comment:
I do like the idea of adding name support here. However, rather than making this optional, I think a better solution would be to follow the existing convention of providing a meaningful default. Following the existing convention, I would recommend using "Mixer" for the default name.